lib/esm/generated/sessionReplay.d.ts

```diff
@@ -586,6 +590,10 @@ export declare type ImageWireframeUpdate = CommonShapeWireframeUpdate & {
      * base64 representation of the image. Not required as the ImageWireframe can be initialised without any base64
      */
     base64?: string;
+    /**
+     * Unique identifier of the image resource
+     */
+    resourceIdentifier?: string;
```

Comment:
hmmm...should we do it optional ?

Reply:
No ta TS expect, but I think it already is (hence the `?` at the end of the name)

within JSON there's array: ```"required": ["type"],``` which I haven't touched for this one

Reply:
the way it should be : `resourceIdentifier: string` and then if you do not want it required (you want it optional) you do not add it in the `required` fields. But I think this field should not be optional. @mquentin what do you think ?

Reply:
> Not a TS expert, but I think it already is (hence the ? at the end of the name)

yes you are correct.

Normally, you generated it with the CLI and it's set as optional because it's not in `required`